lib/classes/game.rb

```diff
@@ -12,59 +12,66 @@ def switch_players
 			@current_player, @other_player = @other_player, @current_player
 		end
 
-		def solicit_move
-			"#{@current_player.name}: Enter a number between 1 and 9."
+		def solicit_move(again = false)
+			unless again
+				puts "#{@current_player.name}: Enter a number between 1 and 9.\n"
+			else
+				puts "#{@current_player.name}: Enter a number between 1 and 9. Please enter a number for an empty cell."
+			end
```

Comment:
If you have an `else` statement use `if`

Reply:
Thanks, resolved with commit [acf4f8bba97343d448e80321113fee525ea50369](https://github.com/adriaanbd/tic-tac-toe/pull/18/commits/acf4f8bba97343d448e80321113fee525ea50369)